opennms-dao/pom.xml

```diff
@@ -177,6 +177,10 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
```

Comment:
No longer needed when we move away from `StringEscapeUtils`.

Reply:
done